parallel_wavegan/utils/utils.py

```diff
@@ -370,11 +371,18 @@ def download_pretrained_model(tag, download_dir=None):
         str: Path of downloaded model checkpoint.
 
     """
-    assert tag in PRETRAINED_MODEL_LIST, f"{tag} does not exists."
-    id_ = PRETRAINED_MODEL_LIST[tag]
     if download_dir is None:
         download_dir = os.path.expanduser("~/.cache/parallel_wavegan")
-    output_path = f"{download_dir}/{tag}.tar.gz"
+    if tag_or_url in PRETRAINED_MODEL_LIST:
+        id_ = PRETRAINED_MODEL_LIST[tag_or_url]
+        output_path = f"{download_dir}/{tag_or_url}.tar.gz"
+    else:
+        # get google drive id from the url link
```

Comment:
Is it possible to check the URL format here?
e.g., `assert "hogehoge" in tag_or_url, "Unknown URL format."`